### ISSUE TITLE
Replace wordnet with any other dictionary

### ISSUE THREAD

**Issue by Nimmi-Ghetia** [superscriptjs/superscript#319]
Can v use any other dictionary in place of wordnet?

**Comment by benhjames** [comment]
It's an interesting idea. I tried deploying SS to AWS Lambda this weekend, and among other reasons, WordNet was the biggest reason why it wasn't as easy as I thought. The unzipped WordNet packages we depend on contribute to about 90Mb of size, which is well over the 50Mb AWS limit.

That isn't to say deploying to AWS isn't impossible, I think I saw someone use a shell script to download WordNet straight to a Lambda's file system as a post upload step. But WordNet certainly makes this harder.

I'd like to evaluate the current usefulness of WordNet and also investigate alternatives for synonym lookup, dictionary definition etc. It's really nice having synonyms and definitions in one, which WordNet provides, but judging from some of the weird results you occasionally get from it, such as `~like` mapping to `cottons for`, there's maybe some improvement there.

A long way of saying, are there any dictionaries in particular you were thinking could be useful? :smiley:

Or perhaps you're more interested in custom concepts like in #317?
